code/modules/research/techweb/nodes/robo_nodes.dm

```diff
@@ -95,7 +95,7 @@
 		"freeformcore_module",
 		"onehuman_module",
 		"purge_module",
-		"ai_power_upgrade"
+		"ai_power_upgrade",
```

Comment:
Well now you have to put this in APC order as it should be.

Reply:
You mean ABC order?

Reply:
yes

Reply:
uhhh... sure